Microsoft.ML.sln

```diff
@@ -928,6 +930,18 @@ Global
 		{5E920CAC-5A28-42FB-936E-49C472130953}.Release-Intrinsics|Any CPU.Build.0 = Release-Intrinsics|Any CPU
 		{5E920CAC-5A28-42FB-936E-49C472130953}.Release-netfx|Any CPU.ActiveCfg = Release-netfx|Any CPU
 		{5E920CAC-5A28-42FB-936E-49C472130953}.Release-netfx|Any CPU.Build.0 = Release-netfx|Any CPU
+		{CFED9F0C-FF81-4C96-8D5E-0436264CA7B5}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{CFED9F0C-FF81-4C96-8D5E-0436264CA7B5}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{CFED9F0C-FF81-4C96-8D5E-0436264CA7B5}.Debug-Intrinsics|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Did you manually add these?  They look wrong to me.

For example, this line should be:

```
  {CFED9F0C-FF81-4C96-8D5E-0436264CA7B5}.Debug-Intrinsics|Any CPU.ActiveCfg = Debug-Intrinsics|Any CPU
```

and the `-netfx` ones should have `-netfx` on the right side too.

This is probably why CI is failing.

Reply:
Autogenerated. I'll fix these up.

---
In reply to: [255266128](https://github.com/dotnet/machinelearning/pull/2470#discussion_r255266128) [](ancestors = 255266128)

Reply:
I copied the lines from Microsoft.ML.Tests/ and swapped in the correct GUID.

---
In reply to: [255266430](https://github.com/dotnet/machinelearning/pull/2470#discussion_r255266430) [](ancestors = 255266430,255266128)